solutions/src/test/java/ru/sberbank/school/task08/KryoManagerTest.java

```diff
@@ -0,0 +1,43 @@
+package ru.sberbank.school.task08;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import ru.sberbank.school.task08.state.GameObject;
+import ru.sberbank.school.task08.state.InstantiatableEntity;
+import ru.sberbank.school.task08.state.MapState;
+import ru.sberbank.school.task08.state.Savable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class KryoManagerTest {
+    private static KryoManager manager;
+    private static MapState<GameObject> mapState;
+    private static String filePath = "C:\\test";
+    private static String fileName = "kryo";
+
+    @Before
+    public void init() {
+        manager = new KryoManager(filePath);
+        manager.initialize();
+        List<GameObject> gameObjectList = new ArrayList<>();
+        gameObjectList.add((GameObject) manager.
+                createEntity(InstantiatableEntity.Type.ENEMY, InstantiatableEntity.Status.KILLED, 0));
+        gameObjectList.add((GameObject) manager.
```

Comment:
&#39;.&#39; should be on a new line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-972847858>*